code/drasil-lang/Language/Drasil.hs

```diff
@@ -190,7 +190,7 @@ module Language.Drasil (
   , scale, shift
   , derUC, derUC', derUC''
   , fund, fund', compUnitDefn, derCUC, derCUC', derCUC''
-  , unitWrapper, getCu, MayHaveUnit(getUnit)
+  , unitWrapper, getCu, MayHaveUnit(getUnit), HasUnit(findUnit)
```

Comment:
This should not be exported. This `findUnit` is a temporary fix as you change things around.